.github/workflows/ci-pipeline.yml

```diff
@@ -26,14 +26,13 @@ jobs:
         if: github.event.schedule != '08 13 * * *'
       - name: Run e2e tests
         run: | 
+          export MONGO_INITDB_ROOT_USERNAME=${{ secrets.MONGO_INITDB_ROOT_USERNAME }} &&\
+          export MONGO_INITDB_ROOT_PASSWORD=${{ secrets.MONGO_INITDB_ROOT_PASSWORD }} &&\
```

Comment:
Nothing wrong with this at all, but I wondered whether we could nicely capture a local mongo DB as part of our docker-compose now to make this easier.. will post more below